CHANGES.txt

```diff
@@ -21,6 +21,10 @@ Changes
 Fixes
 =====
 
+- Fixed the evaluation of JavaScript user defined functions that caused CrateDB
+  to crash because of an unhandled assertion when providing the UDF with
+  EcmaScript 6 arrow function syntax (``var f = (x) => x;``).
```

Comment:
Do you think it's worth mentioning that this only was the case under certain JVM versions?

Reply:
The problem occurs both with OpenJDK and OracleJDK and is only fixed in Java 9. Since we do not support Java 9 yet, I think it is not necessary to mention anything about versions.